frontend/components/flashcard/CardList.tsx

```diff
@@ -40,11 +42,18 @@ export const CardList = ({
 						className="snap-center"
 						key={card.cardID}
 						setCard={setCard}
-						card={{ ...card, deckID: deckID }}
+						card={{ ...card, deckID: deck.deckID }}
 					/>
 				))}
 			</div>
-			<Card card={{ cardID: "", sides: [], deckID: deckID }} />
+			{isAuthorized()?(<Card card={{ cardID: "", sides: [], deckID: deck.deckID }}/>):(<></>)}
 		</div>
 	);
+
+	function isAuthorized(){
+		if(deck.groupRole && GroupRole[deck.groupRole] >= GroupRole.WRITE){
+			return true;
+		}
+		return false;
+	}
```

Comment:
```suggestion

	function isAuthorized(){
		return deck.groupRole && GroupRole[deck.groupRole] >= GroupRole.WRITE
	}
```
if statement not necessary
```suggestion
```
remove function as stated above